playgrounds/geo-javascript/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Can this be shared somehow?

Reply:
What part?

Reply:
The whole file, between the other playgrounds.

Reply:
No they should not rely on out-of-their scope files. They are already using the configuration from `/eslint-config-meilisearch` but they need `.eslintrc` to trigger it in their scope. 

See the structure in this doc: https://turbo.build/repo/docs/handbook/linting/eslint